python/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pylance"
-version = "0.10.16"
+version = "0.10.17"
```

Comment:
I'm incrementing the version now, since the new release process releases the current version *then* increments after.